### PR TITLE
Add some text to filters summary, add Chroma Smooth and Colorspace filter

### PR DESCRIPTION
**Pull request by Nomis101**
I've added the new Chroma Smooth and Colorspace filter to the filters summary. Because there was no text yet for the other filters, I added some. I just copied the explanation texts from HandBrake app, because there is no way I could describe this better in my own words. For the Chroma Smooth filter I've added Bradley's explanation from the Bug. I removed the Rotation filter, because it was removed from the filters tab. Didn't know where to add it instead (I could' find a description of the Overview tab).
Because the filters were not referenced in the table-of-contents, I've added it there.

### CODE REVIEW

File: source/docs/en/latest/technical/filters-summary.markdown [hunk 1]
```diff
@@ -26,5 +25,76 @@ HandBrake contains the following filters:
...
+Denoise
+-------
+
+Denoise reduces or removes the appearance of noise and grain. This can improve compression efficiency and create higher quality video at smaller file sizes. 
+Overly strong Denoise settings may damage picture quality by discarding detail.
+
+NLMeans is a high quality denoise filter with a cost to speed. Use where quality is more important than speed.
+HQDN3D is an adaptive low-pass filter, faster than NLMeans but less effective at preserving fine detail.
+
+The following Denoise tunes are available:
```
Comment (sr55) [inline source/docs/en/latest/technical/filters-summary.markdown:62]: Might want to clarify that these apply to nlmeans

File: source/docs/en/latest/technical/filters-summary.markdown [hunk 2]
```diff
@@ -26,5 +25,76 @@ HandBrake contains the following filters:
...
+Deblock
+-------
+
+Deblock reduces blocky artifacts caused by low quality video compression.
+
+
+Chroma Smooth
+-------------
+
+Reduces chroma noise, rainbows, and other prominent visual artifacts. Useful for resolving various color-related issues, especially with lower resolution content from analog sources, e.g. not great DVD and VHS sources. The blurring/smoothing algorithm is the same as used by the unsharp filter.
```
Comment (sr55) [inline source/docs/en/latest/technical/filters-summary.markdown:81]: "not great"   maybe "poor quality" is better english maybe?

File: source/docs/en/latest/technical/filters-summary.markdown (hunk 1 above)
Comment (Nomis101) [reply source/docs/en/latest/technical/filters-summary.markdown]: Yes, you are right.

File: source/docs/en/latest/technical/filters-summary.markdown (hunk 2 above)
Comment (Nomis101) [reply source/docs/en/latest/technical/filters-summary.markdown]: Did so.

**Comment by bradleysepos** [comment]
Thanks for doing this @Nomis101.

**Comment by sr55** [comment]
@Nomis101  Thanks

You may also want to go onto your github profile settings ->  SSH and GPG Keys -> and add your public key here.  Allows github to show your commits as "verified" :)

**Comment by Nomis101** [comment]
> @Nomis101 Thanks
> 
> You may also want to go onto your github profile settings -> SSH and GPG Keys -> and add your public key here. Allows github to show your commits as "verified" :)

I did so already 2 days ago.

**Comment by sr55** [comment]
odd. Github isn't recognising it. I'll double check the manual incase we also need to add it our side.

**Pull request merged**